src/web/handlers/FlingGestureHandler.ts

```diff
@@ -120,7 +162,7 @@ export default class FlingGestureHandler extends GestureHandler {
     }
   }
 
-  protected onPointerMove(event: AdaptedEvent): void {
+  protected pointerMove(event: AdaptedEvent): void {
```

Comment:
```suggestion
  private pointerMove(event: AdaptedEvent): void {
```

I think this can be `private` instead of `protected`